evms/nethermind.go

```diff
@@ -72,6 +72,19 @@ func (evm *NethermindVM) RunStateTest(path string, out io.Writer, speedTest bool
 	cmd := exec.Command(evm.path, "--input", path,
 		"--trace",
 		"-m") // -m excludes memory
+	if Docker {
+		args := []string{
+			"run",
+			"-v",
+			"/tmp:/tmp/",
+			"nethermind/nethermind",
+			"--nomemory",
+			"--json",
+			"state-test",
```

Comment:
arguments look wrong

Reply:
Yeah I know, there's a lot wrong here... Especially because nethermind doesn't even provide a docker image for nethtest^^